### ISSUE TITLE
data-bind click in IE9

### ISSUE THREAD

**Issue by desnet** [knockout/knockout#1707]
I have:
<ul data-bind="foreach: places">
    <li>
        <span data-bind="text: $data"></span>
        <a href="#" data-bind="click: function(data, event){console.log(data, event, $data)}">Remove</a>
    </li>
</ul>

but in IE9 variables data, event and $data is undefined. Why?

**Comment by brianmhunt** [comment]
@desnet Is this an IE9 specific issue? I would expect it to occur in all browsers.

Why would you use `$data` instead of `data` in this case?

(p.s. you may enjoy the [knockout.punches](http://mbest.github.io/knockout.punches/) library if you are using this pattern a lot)

Cheers

**Comment by desnet** [comment]
yes, this is IE9 specific issue. In IE> 9 and the latest versions of other browsers do not have this problem. I use $data and data. Both of them are not defined.

**Comment by desnet** [comment]
ko.bindingHandlers['test'] = {
        update: function(element, valueAccessor) {
            console.log(valueAccessor);
            console.log(element);
        }
    };

In IE9 data-bind is not working. element, valueAccessor variables is undefined. I use IE11 in document mode IE9, may be in this problem?

**Comment by mbest** [comment]
>  I use IE11 in document mode IE9, may be in this problem?

Probably. This isn't a scenario we normally test.

**Comment by jods4** [comment]
I use KO in a "IE11 in IE9 mode" config on a regular basis and it works. At least with older releases, the project in question is not on the latest release (yet).
